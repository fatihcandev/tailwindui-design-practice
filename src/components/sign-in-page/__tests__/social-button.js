import React from 'react'
import renderer from 'react-test-renderer'

import SocialButton from '../social-button'

describe('SocialButton', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<SocialButton />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
