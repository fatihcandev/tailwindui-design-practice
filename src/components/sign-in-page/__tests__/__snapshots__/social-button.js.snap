// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SocialButton renders correctly 1`] = `
<button
  className="socialButton"
/>
`;
